pyvista/utilities/parametric_objects.py

```diff
@@ -121,32 +119,31 @@ def ParametricBour(**kwargs):
 def ParametricBoy(zscale=None, **kwargs):
     """Generate Boy's surface.
 
-    ParametricBoy generates Boy's surface.
-    This is a Model of the projective plane without singularities.
-    It was found by Werner Boy on assignment from David Hilbert.
+    This is a model of the projective plane without singularities.  It
+    was found by Werner Boy on assignment from David Hilbert.
 
     For further information about this surface, please consult the
     technical description "Parametric surfaces" in the
     "VTK Technical Documents" section in the VTK.org web pages.
 
     Parameters
     ----------
-    zscale : double, optional
-        The scale factor for the z-coordinate.
-        Default is 18, giving a nice shape.
+    zscale : float, optional
+        The scale factor for the z-coordinate.  Default is 18, giving
```

Comment:
This "18" figure seems bogus to me based on experiments. Seems close to 0.1.

Reply:
Removed altogether as it's not specified in VTK docs.